FWIO/RNTuple/test/PreRNTupleInputTest_cfg.py

```diff
@@ -0,0 +1,37 @@
+# The following comments couldn't be translated into the new config version:
+
+# Configuration file for PreRNTupleInputTest 
+
+import FWCore.ParameterSet.Config as cms
+import sys
+
+useOtherThing = False
+if len(sys.argv) > 7:
+  if sys.argv[7] == "useOtherThing":
+    useOtherThing = True
```

Comment:
Would `argparse` be worth it?

Reply:
probably not given this was just a copy from the IOPool/Input tests and adding argsparse would also require change to the .sh file running all the tests.